tests/conftest.py

```diff
@@ -124,3 +127,31 @@ def mock_ocrdataset(tmpdir_factory, mock_image_stream):
             f.write(file.getbuffer())
 
     return str(image_folder), str(label_file)
+
+
+@pytest.fixture(scope="session")
+def mock_svhn_dataset(tmpdir_factory, mock_image_stream) -> None:
+    # get cache path
+    root = tmpdir_factory.mktemp(os.path.join(os.path.expanduser('~'), '.cache', 'doctr', 'datasets'))
```

Comment:
@fg-mindee 
back to a work in progress :sweat_smile: 
Do you have any recommendations to mock the cache dir ?
Otherwise, suggestions are also very welcome for other things :hugs: